src/lib/kdmp-parser.h

```diff
@@ -35,7 +41,7 @@ class KernelDumpParser {
   // File path to the crash-dump.
   //
 
-  const char *PathFile_ = nullptr;
+  std::filesystem::path PathFile_{};
```

Comment:
No need for `{}`

Reply:
Fixed with fe1a51ba9829d9e06efcc9caadf54ddd407ab24b